features/plugins/wpSmush.feature

```diff
@@ -12,26 +12,26 @@ Scenario: WP Smush is installed, network activated, visible on Sidebar and verif
   Then I should visit "wp-admin/network/settings.php?page=wp-smush"
   Then I should see "WP SMUSH PRO"
   Then I should see "SETTINGS"
-  Then I should see "Enable Network wide settings"
-  Then the "wp-smush-networkwide" checkbox should be checked
-  Then I should see "Automatically smush my images on upload"
-  Then the "wp-smush-auto" checkbox should be checked
+  Then I should see "Use network settings for all the sub-sites."

```

Comment:
@harshtrivedi134, can you comment out the last line and create a new one instead of modifying it? We may need to uncomment it again later.

Reply:
@sadam0930  done!